### ISSUE TITLE
need single parameter option specific to card

### ISSUE THREAD

**Issue by a-dam-berger** [nicehash/NiceHashMiner#2767]
The new Extra Commands window does not allow for single parameters that are specific to a gpu. Example: --lhr for the LHR gpu is needed but shouldn't be applied at the algorithm level. Currently the card specific design only allows for splitting of an option but all gpu have to have the option and there isn't an enabled/disabled flag for some lhr option codes.

**Comment by Dohtar1337** [comment]
I am sorry, but I do not understand you fully. Please give me an example of the use case you are trying with.

I guess that you want to have different (example) LHR values for different cards, e.g.:

GPU 1 should have --lhr 1
GPU 2 should have --lhr 2

If you try to set --lhr in the "per GPU" inside Extra Commands, you are left with:

"-**-lhr 1,2**", which is not the desired outcome.

Instead of running two miner consoles at the same time, one next to the other. Is that correct?

To sum up, at the moment, you can only run one --lhr mode at the same time, you would like to run two different modes at the same time for two different GPUs.
